### ISSUE TITLE
contig failing on bubble pinching

### ISSUE THREAD

**Issue by travc** [aquaskyline/SOAPdenovo2#7]
It segfaults very shortly after `Start to pinch bubbles`.

I had seen the error message `Error : Number of vertex is out of range`, but can't seem to get it to reproduce that at the moment.

I'm running from a fresh pull (2015-09-28) and build on Ubuntu 14.04.3 LTS.

**Comment by travc** [comment]
Ok... This may have been due to some issue when compiling.  I have no clue how.
When I rebuilt with `debug=1` and `-j20`, it ended up failing to resolve some symbols which should have clearly been there.  However, when I built with just a single thread (no `-j20`') it linked fine and didn't hit the above bug.

Anyways, maybe there is something with the way you have your Makefiles structured that leads to odd results when making with multiple jobs.  

For what it's worth, I using:
GNU Make 3.81
gcc (Ubuntu 4.8.4-2ubuntu1~14.04) 4.8.4

I'm going to close this, but you might want to look into it.  Sorry, I don't have the time.
